### ISSUE TITLE
Triggering plugin workflow from pull request of core

### ISSUE THREAD

**Issue by dalmijn** [Deltares/hydromt#533]
### Kind of request

Adding new functionality

### Enhancement Description

There already is a workflow in the SFINCS plugin to test against the current main branch of HydroMT.
I myself am working on a workflow for the Wflow plugin that also incorperates some caching.
However, these workflows are only triggered from within the plugin.

Since the release of 0.8.0 of HydroMT, quite a few changes have been made, while the Wflow plugin has not been tested all that much since then. To maybe not create large gaps between the core and the plugin (if for whatever reason not much work is being done on the plugins) it would be nice (if at all possible) to trigger the testing workflows of the plugins (vs the main branch of HydroMT) when a pull request is merged into the main branch of HydroMT. This would make is way easier to pinpoint where bugs and errors originate from when these occur within the plugins.

### Use case

Working on plugins.

**Comment by savente93** [comment]
Just writing about the outcome of an earlier conversation we had about this.  This will be difficult until core improves its version system. Especially since PRs are often merged in bursts, creating contention for the tests to be run (and also doing unnecessary work). Until the versioning of core is improved, a much better ROI option is to run the tests against core-dev on a schedule, something like daily at midnight. Especially since it might take a couple of days to work on flagged issues anyway, drift of up to a day, seems acceptable. I'll be happy to help any plugin devs that want it setup the scheduled testing, but otherwise I'll be closing this as not planned. Please reopen it if you disagree, at which point we'll have to inventories what needs to be done to unblock this and get an idea for when that might be done.
